src/main/scala/com/ubirch/webui/models/keycloak/member/ClaimTransaction.scala

```diff
@@ -42,23 +42,27 @@ class ClaimTransaction(device: MemberResourceRepresentation, prefix: String, tag
 
     device.leaveGroup(unclaimedGroup)
 
+    //update password
```

Comment:
The whole method is really long and therefore difficult to read. I some time heard/ learned that a method is not to encompass more than 5 lines. When you're applying this rule, you can read code like a book. I really like this approach. Then you don't need to add comments here and there. :) What do you think of that idea?